main.go

```diff
@@ -29,6 +32,36 @@ func main() {
 		os.Exit(1)
 	}
 
+	results, err := run(params)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Test Service Error: %v\n", err)
```

Comment:
```suggestion
		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
```
"Test service" would refer to the service implementation provided by the SDK, so there are lots of error conditions below that are not a "test service error".